utils/init.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# run this from the project root
+# if you do not have permission to run it, remember to chmod +x the script file
+url='https://developer.x-plane.com/wp-content/plugins/code-sample-generation/sdk_zip_files/XPSDK411.zip'
+mkdir -p ./lib
+curl ${url} > sdk.zip
+unzip sdk.zip -d ./lib
+rm sdk.zip
```

Comment:
... it could replace all of this, probably.

Reply:
Ah wait hold on a minute, this reminds me...

Someone added x-plane-sdk wrap to meson's wrapdb last year. `meson wrap install x-plane-sdk` and use

```meson
deps = [
    dependency('xplm'),
    dependency('xpwidgets')
]
```